server/controllers/user_controller.js

```diff
@@ -24,7 +26,7 @@ class UserController {
         is_admin,
       } = req.body;
       is_mentor = false;
-      is_admin = false;
+      is_admin = true;
```

Comment:
Why are you setting is_admin to true here, when are you creating the script for adding an admin?

Reply:
I have resolved this